public/data/rajiv.json

```
@@ -0,0 +1,17 @@
+{
+    "name": "Rajiv Tiwari",
+    "bio": "React native developer",
+    "avatar": "https://media-exp1.licdn.com/dms/image/C4D03AQH45Ji-uNSU1w/profile-displayphoto-shrink_100_100/0/1625467516815?e=1637193600&v=beta&t=7Zn-lvnCqSHXMQd9OdYMSDPYqhalxAKFQkMg7flYzIU",
```

```suggestion
    "avatar": "https://github.com/rajivransom.png",
```
We are using a standard format for avatar URLs to ensure images render.